packages/frontend/src/pages/scaling-detailedTvl/view/DetailedTvlPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import {
+  Chart,
+  Footer,
+  FooterProps,
+  Navbar,
+  NavbarProps,
+} from '../../../components'
+import { About } from '../../../components/About'
+import { TvlHeader } from '../../../components/header/TvlHeader'
+import { ScalingNavigationTabs } from '../../../components/navigation-tabs/ScalingNavigationTabs'
+import { OtherSites } from '../../../components/other-sites/OtherSites'
+import { PageContent } from '../../../components/PageContent'
+import { DetailedTvlView, DetailedTvlViewProps } from './DetailedTvlView'
+
+export interface DetailedTvlPageProps {
+  tvl: string
+  tvlWeeklyChange: string
+  navbar: NavbarProps
+  footer: FooterProps
+  showDetailedTvl: boolean
+  showActivity: boolean
+  detailedTvlView: DetailedTvlViewProps
+}
+
+export function DetailedTvlPage(props: DetailedTvlPageProps) {
+  return (
+    <>
+      <Navbar {...props.navbar} />
+      <PageContent>
+        <ScalingNavigationTabs
+          showActivity={props.showActivity}
+          showDetailedTvl={props.showDetailedTvl}
+          selected="detailed"
+        />
+        <main>
+          <TvlHeader tvl={props.tvl} tvlWeeklyChange={props.tvlWeeklyChange} />
+          <Chart detailedTvlEndpoint={'halucinate_data'} type={'detailedTvl'} />
```

Comment:
maybe halucinate_data can be renamed to mocked_data?

Reply:
It's just a placeholder string, the content does not matter as it's going to be removed